paddle/fluid/operators/lookup_table_op.cc

```diff
@@ -85,6 +91,44 @@ or not. And the output only shares the LoD information with input Ids.
   }
 };
 
+class ConcatRowsOpMaker : public framework::OpProtoAndCheckerMaker {
```

Comment:
I think this name is not so good and maybe we just need a python layer but not a new operator

Reply:
Thanks for your review @jacquesqiao 
The parameters' meaning of ConcatRowsOp are different with LookUpTableOp, and the two operations' function is also different, so I define the new operation(ConcatRows). And I have added some annotation in the code.

Is your meaning that ConcatRowsOp is unnecessary?